src/test/java/Homework21.java

```diff
@@ -0,0 +1,51 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+
+    String NewPlaylistName = "Renamed Playlist";
+
+    @Test
+    public void renamePlayList() throws InterruptedException {
+
+        String updatedPlaylistMsg = "Updated playlist \"Renamed Playlist.\"";
+
+        //Login
+        provideEmail("demo@class.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(2000);
+        //double click
+        doubleClickPlaylist();
+        Thread.sleep(2000);
+        //Enter New Name for Playlist
+        enterNewPlaylistName();
+        Thread.sleep(2000);
+        //Assert
+        Assert.assertEquals(getRenamePlaylistSuccessMsg(), updatedPlaylistMsg);
+    }
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.